plugins/plugin-k8s/src/test/k8s/get-namespaces-with-watch.ts

```diff
@@ -141,5 +141,6 @@ describe('electron watch namespace', function (this: common.ISuite) {
     createIt()
     deleteIt()
```

Comment:
> lgtwm, except for typecov issue in travis?

In my previous try, the typecov regression is caused by `deleteIt = deleteNS.bind(this, kubectl)`, and `Function.bind` returns `type: any` here.

 I work around this by using the delete function in k8s/test/lib/util  instead of re-using the delete-from-url in this test. Not sure if this is idea though...

Reply:
Maybe specify a type on the result of the bind? Seems weird that typescript doesn't handle bind.

Reply:
I think we need to enable `strict` to make it compiled with `--strictBindCallApply,` which needs us to eliminate the errors first... 
In my third try, I worked around by adding types manually . However, with `--strictBindCallApply`, we should get the types for free.
(--strictBindCallApply: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-2.html)